dataflow/snippets/pom.xml

```diff
@@ -156,6 +156,24 @@
       <version>${iceberg.version}</version>
     </dependency>
 
+    <!-- Kafka -->
+    <dependency>
+      <groupId>org.apache.beam</groupId>
+      <artifactId>beam-sdks-java-io-kafka</artifactId>
+      <version>${apache_beam.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.apache.kafka</groupId>
+      <artifactId>kafka-clients</artifactId>
+      <version>3.8.0</version>
+    </dependency>
+    <dependency>
+      <groupId>org.testcontainers</groupId>
+      <artifactId>kafka</artifactId>
+      <version>1.20.0</version>
```

Comment:
nit: is there a dependency between these two versions? should we set kafka client version as a variable?

Reply:
As far as I know, there's no dependency. The kafka-clients module is for sending messages to Kafka, the testcontainers module is to spin up a Kafka container (these are both for the IT). So they only interact via Kafka which should be decoupled.

Reply:
Thank you. Since both these dependencies seem to be used for testing, please add the `test` scope to the `kafka-clients` dependency.

Reply:
done